### ISSUE TITLE
Documenting the process

### ISSUE THREAD

**Issue by egekorkan** [w3c/wot-resources#3]
With #1 and #2 in flight, we should document how this process happens. Ideally here instead of duplicating it in every repo

**Comment by egekorkan** [comment]
My proposal is to say that with REC publication transition, the documents MUST be copied here without changing anything (name and content).

**Comment by danielpeintner** [comment]
To avoid any inconsistency, does it make sense that once the content is copied over to `wot-resources` the original content is to be removed and simply replaced by a pointer to the _new_ location?

**Comment by egekorkan** [comment]
@danielpeintner sorry for not replying. You mean deleting https://github.com/w3c/wot-thing-description/tree/main/publication/ver11/7-resources right ?

**Comment by danielpeintner** [comment]
Yes, exactly. I believe it is a bit dangerous having the same content in several places without people knowing what is the correct place. Having links or GitHub tags pointing to a certain revision makes more sense.
